tests/crawl/test_crawl.py

```
@@ -0,0 +1,38 @@
+  "Head-Neck-PET-CT"
+])
+def test_shared_datadir_datasets(data_paths, dataset_key) -> None:
+  """
+  Test that each dataset key exists in the copied shared dataset directory.
+  """
+  rprint(f"Checking dataset: {dataset_key}")
+  assert dataset_key in data_paths, f"Dataset key {dataset_key} not found"
+  assert data_paths[dataset_key].exists(), f"Dataset {dataset_key} not found"
+
+
+  db = crawl(
```

Ruff (0.8.2)

Local variable `db` is assigned to but never used

Remove assignment to unused variable `db`

(F841)